README.md

```diff
@@ -18,14 +18,15 @@ NOTE: you must have write permissions to the repository in question. If you're u
 
 ## Dependencies on other GitHub Actions
 
-* [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into GitHub Actions agent.
```

Comment:
I think this is correct. 
Github Action name and what it does.

Apart from Amit's comment on extra space, looks good to me.